lapack/build.sh

```diff
@@ -2,10 +2,11 @@ mkdir build
 cd build
 
 cmake \
-    -DCMAKE_INSTALL_PREFIX=$PREFIX \
-    -DBUILD_TESTING=OFF \
-    -DBUILD_SHARED_LIBS=ON \
-    ..
+  -DCMAKE_INSTALL_PREFIX=$PREFIX \
+  -DBUILD_TESTING=OFF \
+  -DBUILD_SHARED_LIBS=ON \
+  -DLAPACKE=ON \
```

Comment:
I believe this is for the C interface to LAPACK. Am a bit surprised this wasn't already enabled.